package.json

```diff
@@ -100,7 +100,7 @@
     "aws-lambda": "^1.0.5",
     "copyfiles": "^2.3.0",
     "cross-env": "^5.2.0",
-    "electron": "22.3.12",
+    "electron": "26.6.10",
```

Comment:
@FridmasterDyna 
Also, this will auto update explorook for a few users and will break their epxplorook app

I recommend, at least, releasing a version that uses electron 22 and does not auto update, and inform those users they need to use that version